src/tests/ftest/util/telemetry_utils.py

```diff
@@ -288,14 +451,14 @@ def check_container_metrics(self, open_count=None, active_count=None,
         """
         errors = []
         expected = {
-            "engine_container_ops_open_total": open_count,
-            "engine_container_ops_open_active": active_count,
-            "engine_container_ops_close_total": close_count,
-            "engine_container_ops_destroy_total": destroy_count,
+            "engine_pool_ops_cont_open": open_count,
+            "engine_pool_container_handles": active_count,
+            "engine_pool_ops_cont_close": close_count,
+            "engine_pool_ops_cont_destroy": destroy_count,
         }
         data = self.get_container_metrics()
         for host in data:
-            for name in self.ENGINE_CONTAINER_METRICS:
+            for name in expected.keys():
```

Comment:
(pylint-consider-iterating-dictionary) Consider iterating the dictionary directly instead of calling .keys()